source/SpinalHDL/Developers area/mill support.rst

```diff
@@ -0,0 +1,47 @@
+Build through Mill
+==================
+
+SpinalHDL itself can be built by Mill build tools, it could compile/test/publishLocal the existing modules.
+Build through mill can be much faster than sbt, which is useful while debugging.
```

Comment:
```suggestion
SpinalHDL itself can be built with Mill. It can compile/test/publishLocal the existing modules.
Build through mill can be much faster than Sbt, which is useful while debugging.
```